modules/ols-enabling-cluster-interaction.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-cluster-interaction_{context}"]
+= Enabling cluster interaction
+
+Modify the `OLSConfig` custom resource to enable the cluster interaction feature.
+
+[NOTE]
+====
+Cluster interaction is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR file.
+
+* You have configured the large language model (LLM) provider.
+
+* You have installed the {ols-long} Operator.
+
+.Procedure 
+
+. In the {ocp-product-title} web console, click *Operators* -> *Installed Operators*. 
```

Comment:
??
```suggestion
. In the {product-title} web console, click *Operators* -> *Installed Operators*. 
```

Reply:
we use ocp-product-title in the standalone doc set.